src/constants.ts

```diff
@@ -87,10 +99,16 @@ export class Constants {
     public static IoTHubAIDeleteModuleDoneEvent = "AZ.Module.Delete.Done";
     public static IoTHubAIGetModuleStartEvent = "AZ.Module.Get.Start";
     public static IoTHubAIDGetModuleDoneEvent = "AZ.Module.Get.Done";
+    public static IoTHubAIUpdateDistributedSettingStartEvent = "AZ.DistributedTracing.Update.Start";
+    public static IoTHubAIUpdateDistributedSettingDoneEvent = "AZ.DistributedTracing.Update.Done";
+    public static IoTHubAILoadDistributedTracingSettingStartEvent = "AZ.LoadDistributedTracingSetting.Start";

```

Comment:
Better to be `AZ.LoadDistributedTracingSettingTree.Start`

Reply:
fixed